ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.helpers.js

```diff
@@ -201,16 +201,18 @@ export function cancelAllCveExceptions() {
 
     cy.request({ url: '/v2/vulnerability-exceptions', auth }).as('vulnExceptions');
 
-    cy.get('@vulnExceptions').then((res) => {
-        res.body.exceptions.forEach(({ id, expired }) => {
-            if (!expired) {
-                cy.request({
-                    url: `/v2/vulnerability-exceptions/${id}/cancel`,
-                    auth,
-                    method: 'POST',
-                });
-            }
-        });
+    return cy.get('@vulnExceptions').then((res) => {
+        return Promise.all(
+            res.body.exceptions.map(({ id, expired, requester }) => {
+                return requester?.name === 'ui_tests'
+                    ? cy.request({
+                          url: `/v2/vulnerability-exceptions/${id}`,
+                          auth,
+                          method: 'DELETE',
+                      })
+                    : Promise.resolve();
+            })
+        );
```

Comment:
An unrelated change, but helpful when debugging and cleans up the test implementation a bit:

    1. Instead of fire and forget, we return promises to wait until all deletions have completed
    3. Instead of canceling all requests, instead we delete any request that was created by the e2e test user